theme/js/script.js

```diff
@@ -150,4 +152,23 @@ function validateForm() {
             requiredField.siblings('label').addClass('invalid');
         }
     });
-}
\ No newline at end of file
+}
+
+function getFormData() {
+    $.getUrlParam = function(name){
```

Comment:
Move this entire function outside of getFormData() so it can be used in other functions at a later date.

Reply:
done